README.md

```diff
@@ -383,27 +383,32 @@ items are added to the top of the list by other users in between requests.
 To do this, implement the `afterInfinityModel` hook as follows:
 
 ```js
-export default Ember.Route.extend({
-  _minId: undefined,
-  _minUpdatedAt: undefined,
-
-  infinity: service(),
+import Route from '@ember/routing/route';
+import InfinityModel from 'ember-infinity/lib/infinity-model';
 
-  model() {
-    return this.infinity.model("post", {}, {
-      min_id: '_minId',
-      min_updated_at: '_minUpdatedAt'
-    });
+const ExtendedInfinityModel = InfinityModel.extend({
+  buildParams() {
+    let params = this._super(...arguments);
+    params['min_id']: get(this, '_minId'); // where `this` is the infinityModel instance
+    params['min_updated_at']: get(this, '_minUpdatedAt');
+    return params;
   },
-
-  afterInfinityModel(posts, infinityModel) {
+  afterInfinityModel(posts) {
```

Comment:
this PR makes `afterInfinityModel` something you put on your extended infinity model.  Previously this was always pulled off the route; however, for our service oriented approach, I _think_ this is an ok API (although requires a bit more code)